examples/rest_api/README.md

```diff
@@ -0,0 +1,31 @@
+                    Copyright 2021 ABSA Group Limited
+                  
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+                You may obtain a copy of the License at
+               http://www.apache.org/licenses/LICENSE-2.0
+            
+     Unless required by applicable law or agreed to in writing, software
+       distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+                      limitations under the License.
+
+# Examples of Enceladus rest-api calls
+
+Main purpose of the api calls examples presented here is support of developers from another projects. 
+
+
+There are 3 import files
+
+- `01-Local_IntellijJ Tomcat.postman_environment.json` - contains all variables used in API calls
+- `02-Enceladus.rest-api.v2.postman_collection.json` - define all rest-api v2 compatible calls
+- `03-Enceladus.rest-api.v3.postman_collection.json` - define all rest-api v3 compatible calls
+
+## Setup
+
+### Install Postman
+Download and install Postman from https://www.postman.com/downloads.
+
+### Import into Postman
+Import variables first. Rest of files depends on your needs. 
```

Comment:
I had little issue to find the proper import method (Import button too big 🤣 ), definitely harder then downloading postman, which has its link mentioned. Maybe a little hint to use the button on the main page.... 👼